src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -1349,4 +1351,119 @@ private static int indexOfSameAllele(final VariantContext vc, final Allele allel
 
         return -1;
     }
+
+    /**
+     * Add chromosome Counds (AC, AN and AF) to the VCF header lines
```

Comment:
Is chromosome counts established nomenclature that I'm not aware of?  It seems like allele counts is more accurate to me since this is all per allele.